### ISSUE TITLE
Jest snapshot error with NavDropdown

### ISSUE THREAD

**Issue by ian-s-mcb** [react-bootstrap/react-bootstrap#3754]
## Describe the bug

Jest test snapshots produce an error when rendering NavDropdown components.

## To Reproduce

1. Create component that renders a NavDropdown inside a Nav component
1. Create a Jest test that uses `react-test-renderer`  `.create()` function.
1. Run Jest tests

## Reproducable Example

https://codesandbox.io/s/mm1vn5n4j9?from-embed

## Expected behavior

A properly created Jest snapshot and no error message.

## Screenshot
![screenshot-react-bootstrap_navdropdown_jest_snapshot_error_2019-05-07](https://user-images.githubusercontent.com/2540024/57320773-aaf06a80-70cd-11e9-97d9-9e1784065c55.png)

## Environment (please complete the following information)

- Operating System: arch linux
- Browser, Version: Chrome 72
- React-Bootstrap Version: 1.0.0-beta.8

**Comment by ian-s-mcb** [comment]
Thanks for answer, unfortunately, I upgraded my testing libraries and got the same error.

```bash
$ yarn list --pattern babel-jest jest react-test-renderer
yarn list v1.15.2
warning Filtering by arguments is deprecated. Please use the pattern option instead.
├─ babel-jest@24.8.0
├─ jest@24.8.0
└─ react-test-renderer@16.8.6
```

I tried the newest versions of my testing libraries using a Gatsby JS app, instead of `create-react-app`. Unfortunately, because of https://github.com/facebook/create-react-app/issues/6756 `create-react-app` can't be used with the newest version of `babel-jest` so it's stuck at version: `4.7.1`

This [GitLab repo](https://gitlab.com/ian-s-mcb/rb-jest-bug) has the app I used with the newest versions of my testing libraries. The `README.md` of that repo has the exact steps to recreate the app from scratch.
